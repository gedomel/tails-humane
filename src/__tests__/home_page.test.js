import React from 'react';
import renderer from 'react-test-renderer';
import HomePage from '../Components/home_page';

/**
 * Test Method to ensure that the cat cards render correctly.
 */
it('Render Cat HomePage Correctly', () => {
  const tree = renderer.create(<HomePage species="Cat" />).toJSON();
  expect(tree).toMatchSnapshot();
});

/**
 * Test Method to ensure that the dog cards render as expected.
 */
it('Render Dog HomePage Correctly', () => {
  const tree = renderer.create(<HomePage species="Dog" />).toJSON();
  expect(tree).toMatchSnapshot();
});
