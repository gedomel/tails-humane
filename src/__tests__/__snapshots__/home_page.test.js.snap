// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Cat HomePage Correctly 1`] = `
<div
  className="HomePage"
>
  <div
    className="animal_display"
  />
</div>
`;

exports[`Render Dog HomePage Correctly 1`] = `
<div
  className="HomePage"
>
  <div
    className="animal_display"
  />
</div>
`;
